src/api/SFA.DAS.Provider.Events.Api/Controllers/LearnersController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Web.Http;
+using MediatR;
+using NLog;
+using SFA.DAS.Provider.Events.Application.Submissions.GetLatestLearnerEventByStandardQuery;
+
+namespace SFA.DAS.Provider.Events.Api.Controllers
+{
+    public class LearnersController : ApiController
+    {
+        private IMediator _mediator;
+        private ILogger _logger;
+
+        public LearnersController(IMediator mediator, ILogger logger)
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+
+        [Route("api/v2/learners", Name = "GetLatestLearnerEventByStandard")]
+        [HttpGet]
+        public async Task<IHttpActionResult> GetLatestLearnerEventByStandard(long uln, long sinceEventId = 0)
```

Comment:
The name of the method makes it sound like we are doing it by standard, or that there should be a standard code as a parameter. Can you rename it to either ...ForStandards or leave the standard part out?